lib/uploader/uploader.ts

```diff
@@ -474,10 +474,10 @@ export class Uploader {
 			// We can cast here as we handled system entries in the if above
 			const file = fileHandle as File
 
-			// If manually disabled or if the file is too small
-			// TODO: support chunk uploading in public pages
+			const supportsPublicChunking = getCapabilities().dav?.public_shares_chunking ?? false
```

Comment:
this breaks the tests, the code itself is fine but the tests are (already before) broken.

Fix here would be:
```suggestion
			const supportsPublicChunking = getCapabilities()?.dav?.public_shares_chunking ?? false
```

But probably should be fixed in tests